client-js/app/elements/labrad-plot.ts

```diff
@@ -1126,49 +1316,56 @@ export class Plot extends polymer.Base {
 
 
   /**
+   * Helper function to get mouse position in the coordinates of the svg plot
+   * area. The d3.mouse function returns coordinates relative to the full html
+   * element, so we must account for the margins. We also clip the coordinates
+   * to the available plot area.
+   */
+  private mousePositionClipped(offset: number): number[] {
+    const [x, y] = d3.mouse(this);
+    return [
+      clip(x - this.margin.left, 0, this.width + offset),
+      clip(y - this.margin.top - PLOT_TOP_OFFSET, 0, this.height + offset)
+    ];
+  }
+
+
+  /**
    * Zoom into a selected rectangular region on the graph.
    */
-  private drawZoomRectangle(): void {
+  private drawRectangle(rect: HTMLElement): Promise<RectangleBound> {
```

Comment:
Didn't this change already happen in a previous PR? Not sure why it's showing up here too.